policyengine_us/parameters/gov/states/nc/ncdhhs/scca/age/limit/non_disabled.yaml

```
@@ -0,0 +1,10 @@
+description: North Carolina limits its Subsidized Child Care Assistance program to non-disabled children younger than this age.
```

Otherwise sounds like only for non-disabled children
```suggestion
description: North Carolina limits its Subsidized Child Care Assistance program to children younger than this age, if they do not have a disability.
```